docs/src/notes.md

```diff
@@ -0,0 +1,28 @@
+# Notes
+
+## Using Solvers and Other Additional Packages
+Certain solvers may require a license or an additional installation. Since we want to test the tutorials using Travis CI and run them in the browser using Binder, we should always try to go for the solver which works out of the box with just a Package install. In case this is not possible, do not add the tutorial to the `runtests.jl` file and add a line that it does not work with Binder. Note that this issue is not specific to solvers and may arise with other Julia packages as well.
```

Comment:
Stick to 80 character limit OR use one english sentence per text file line. For markdown I prefer the latter.